fusor-ember-cli/app/styles/custom.scss

```diff
@@ -661,7 +661,7 @@ table.fusor-table {
 }
 
 .input-ose {
-  width: 20px;
+  width: 25px;
```

Comment:
This is so 3 digits (ex 100) will fit in the input field. Previously, with 20px, only 2 digits fit.